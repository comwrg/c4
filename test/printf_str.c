#include <stdio.h>

int main() {
    printf("heiheihei\n");
    return 2333;
}
