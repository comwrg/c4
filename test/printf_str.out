heiheihei
exit(2333)
